src/SignalRServiceExtension/TriggerBindings/ServerlessHub.cs

```diff
@@ -66,13 +70,35 @@ protected ServerlessHub(IServiceHubContext hubContext = null, IServiceManager se
         /// <summary>
         /// Return a <see cref="SignalRConnectionInfo"/> to finish a client negotiation.
         /// </summary>
-        protected SignalRConnectionInfo Negotiate(string userId = null, IList<Claim> claims = null, TimeSpan? lifeTime = null)
+        [Obsolete("Please use async version instead.")]
+        protected SignalRConnectionInfo Negotiate(string userId = null, IList<Claim> claims = null, TimeSpan? lifeTime = null, HttpContext httpContext = null)
         {
-            return new SignalRConnectionInfo
+            return NegotiateAsync(userId, claims, lifeTime, httpContext).Result;
+        }
+
+        /// <summary>
+        /// Return a <see cref="SignalRConnectionInfo"/> to finish a client negotiation.
```

Comment:
how about:
Gets <see cref="SignalRConnectionInfo"/> for SignalR hub connections to connect to Azure SignalR Service